### PR TITLE
refactor: use @linx-impulse/commons-js ajax function to make http requests

### PR DESCRIPTION
**Pull request by andrewcpacifico**
Removes the `make-request` implementation from lib, and uses the already implemented ajax fuction from https://github.com/chaordic/commons-js.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,36 +1,27 @@
...
-    const method = 'GET';
-    const {
-      page,
-      source,
-      deviceId,
-      showLayout,
-    } = options || {};
+    if (!source) {
+      return Promise.reject(new TypeError('source is required to get banners'));
+    }
```
Comment (IvaProtasio) [inline src/index.js]: Adiciona o UserId

File: src/index.js [hunk 2]
```diff
@@ -1,36 +1,27 @@
...
 
-    const params = filterEmptyFields({
-      page, source, deviceId, showLayout,
+    return new Promise((resolve, reject) => {
+      ajax({
+        url: `${config.server.baseUrl}${config.server.recommendationUrl}`,
+        params: { deviceId, page, source, showLayout },
```
Comment (IvaProtasio) [inline src/index.js]: Adiciona o userId

Comment (andrewcpacifico) [reply src/index.js]: Vou colocar numa outra PR.

File: src/index.js (hunk 1 above)
Comment (andrewcpacifico) [reply src/index.js]: Adicionei numa outra PR: #8

**Pull request merged**